catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/Context.java

```diff
@@ -30,6 +31,8 @@
 
     private Optional<Metacard> parentMetacard = Optional.empty();
 
+    private final AtomicBoolean isParentDirty = new AtomicBoolean(false);
```

Comment:
The term dirty does not provide any meaningful information in my opinion.  Why is it dirty?  What needs to be done to make it "clean"?